WooCommerce/Classes/ViewRelated/Products/Categories/ProductCategoryListViewController.swift

```diff
@@ -9,6 +9,7 @@ import WordPressUI
 ///
 final class ProductCategoryListViewController: UIViewController, GhostableViewController {
 
+    @IBOutlet var contentStackView: UIStackView!
```

Comment:
Can we make this `private`?

Reply:
Yup - updated in 47a2f52497cc54ffe02c5f64002cf180dbb163b2.